src/main/java/com/uber/cadence/internal/worker/ActivityWorker.java

```diff
@@ -175,7 +181,18 @@ public void handle(PollForActivityTaskResponse task) throws Exception {
 
       propagateContext(task);
 
-      try {
+      // Set up an opentracing span
```

Comment:
ditto. We don't want to change any of the client code to support a specific context propagator.